File: Application Dev for Mobile Devices/Apps/Lienzo/app/src/main/java/com/example/lienzo/MainActivity.java
package com.example.lienzo;

import android.os.Bundle;
import android.app.Activity;

public class MainActivity extends Activity {

    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Lienzo l = new Lienzo(this);
        setContentView(l);
    }
}

File: Application Dev for Mobile Devices/Apps/KotlinParte1/app/src/main/java/com/example/kotlinparte1/MainActivity.kt
package com.example.kotlinparte1

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.myTextView
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        myTextView.setText("Hello World")
    }
}